WordPressKitTests/RemoteTestCase.swift

```diff
@@ -134,15 +146,20 @@ extension RemoteTestCase {
                 return HTTPStubsResponse(error: notConnectedError)
             }
 
-            let stubPath = OHPathForFile(files[callCounter], type(of: self))
+            guard let stubPath = OHPathForFile(files[callCounter], type(of: self)) else {
+                XCTFail("Could not find file at path '\(files[callCounter])'.")
+                let error = NSError(domain: "RemoteTestCase", code: 0, userInfo: nil)
+                return HTTPStubsResponse(error: error)
+            }
```

Comment:
Had to fail in this way, also returning an error because we're inside a closure and the file path is computed based on the `callCounter` value.